llvm/lib/SYCLLowerIR/SanitizeDeviceGlobal.cpp

```diff
@@ -50,6 +50,10 @@ static bool instrumentDeviceGlobal(Module &M) {
     if (!isDeviceGlobalVariable(G) || !hasDeviceImageScopeProperty(G))
       continue;
 
+    // Skip instrumenting on "__AsanKernelMetadata" etc.
+    if (G.getName().starts_with("__Asan"))
```

Comment:
are there other variables besides `__AsanKernelMetadata`?

Reply:
Not sure, but we used to use "__Asan" as an internal prefix.